includes/rest-api/class-wp-job-manager-models-settings.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Declaration of our Settings Model
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Models_Settings
+ */
+class WP_Job_Manager_Models_Settings extends WPJM_REST_Model_Declaration_Settings
```

Comment:
I was thinking it would suffice, as long as the override of `WPJM_REST_Model_Declaration_Settings::get_settings` returns a similarly shaped array. Maybe this is a documentation thing?